tests/questions/MultipleChoiceNextPreviousTest.php

```diff
@@ -58,20 +58,18 @@ public function testNextPrevious()
             $lis[0]->click();
 
             // Click next.
-            $submit = self::$webDriver->findElement(WebDriverBy::id('ls-button-submit'));
-            $submit->click();
+            $submit = self::$webDriver->next();
 
             // Click previous..
-            $prev = self::$webDriver->findElement(WebDriverBy::id('ls-button-previous'));
+            $prev = self::$webDriver->findElement(WebDriverBy::cssSelector('[value="moveprev"]'));
             $prev->click();
             sleep(1);  // TODO: Does not work without this.
 
             // Click next.
-            $submit = self::$webDriver->findElement(WebDriverBy::id('ls-button-submit'));
-            $submit->click();
+            $submit = self::$webDriver->next();
 
             // Click previous..
-            $prev = self::$webDriver->findElement(WebDriverBy::id('ls-button-previous'));
+            $prev = self::$webDriver->findElement(WebDriverBy::cssSelector('[value="moveprev"]'));
```

Comment:
Unsure of way cssSelector work if there are multiple button, maybe need '[value="moveprev"]:first'

This was not an issue with core template since we use only one button.